src/lib.rs

```diff
@@ -48,12 +49,28 @@ impl Build {
         self,
         crate_name: &str,
         static_dir_path: impl AsRef<Path>,
-        build_dir_path: impl AsRef<Path>,
+        build_dir_path: Option<impl AsRef<Path>>,
```

Comment:
Now ok it's optional but I think you can do better. Think about the builder pattern